packages/amplication-client/src/Project/ProjectList.scss

```diff
@@ -0,0 +1,33 @@
+@import "../style/index.scss";
+
+.project-list {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 8px;
+  width: 248px;
+  height: 656px;
+
+  &__items {
+    width: 248px;
+    height: auto;
+
+    .inner-tab-link {
+      width: 100%;
+      height: 24px;
+      border-radius: var(--small-border-radius);
+      padding: 5px 8px;
+      gap: 8px;
+  
+      .amp-icon {
+        font-size: var(--title1-font-size);
+        margin-right: var(--default-spacing-small);
+      }
+    }
+  }
```

Comment:
I don't expect to see any style specific for this component. 
Since you are using the InnerTabLink - all styles should be inherited from it. 
I also talked with @alexbass86 about it - and he expects that all places where we use the InnerTabLink to get the new look as used in the new design